### ISSUE TITLE
POSIX::tmpnam() is deprecated

### ISSUE THREAD

**Issue by eserte** [ironcamel/POE-Component-MessageQueue#8]
Running the test suite with bleadperl (e.g. 5.23.8) shows the following warnings:
```
...
Calling POSIX::tmpnam() is deprecated at t/lib/POE/Component/MessageQueue/Test/EngineMaker.pm line 29.
t/07_send_disconnect_send.t .. ok
Calling POSIX::tmpnam() is deprecated at t/lib/POE/Component/MessageQueue/Test/EngineMaker.pm line 29.
t/08_removed_file.t .......... ok
Calling POSIX::tmpnam() is deprecated at t/lib/POE/Component/MessageQueue/Test/EngineMaker.pm line 29.
t/09_ack_auto.t .............. ok
...
```

**Comment by ironcamel** [comment]
Thanks, I will try to fix this issue this week.

**Comment by eserte** [comment]
With perl 5.25.1 and later this is now a compilation error:
```
Unimplemented: POSIX::tmpnam(): use File::Temp instead at /usr/perl5.25.1p/lib/5.25.1/amd64-freebsd/POSIX.pm line 187.

#   Failed test 'use POE::Component::MessageQueue::Test::EngineMaker;'
#   at t/05_storage_api.t line 29.
#     Tried to use 'POE::Component::MessageQueue::Test::EngineMaker'.
#     Error:  Unimplemented: POSIX::tmpnam() at t/lib/POE/Component/MessageQueue/Test/EngineMaker.pm line 23.
# BEGIN failed--compilation aborted at t/lib/POE/Component/MessageQueue/Test/EngineMaker.pm line 23.
# Compilation failed in require at t/05_storage_api.t line 29.
# BEGIN failed--compilation aborted at t/05_storage_api.t line 29.
Undefined subroutine &main::engine_names called at t/05_storage_api.t line 33.
BEGIN failed--compilation aborted at t/05_storage_api.t line 35.
# Looks like your test exited with 2 just after 4.
t/05_storage_api.t ........... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 123/126 subtests 
...
```

**Comment by ironcamel** [comment]
Thanks for the reminder

**Comment by ironcamel** [comment]
This is fixed in #9 and I just pushed it to CPAN
